nexus/db-model/src/utilization.rs

```diff
@@ -57,39 +57,15 @@ impl From<SiloUtilization> for views::Utilization {
     }
 }
 
-#[derive(Debug, Clone, Serialize, Deserialize)]
-pub struct Ipv4Utilization {
-    pub allocated: u32,
-    pub capacity: u32,
-}
-
-#[derive(Debug, Clone, Serialize, Deserialize)]
-pub struct Ipv6Utilization {
-    pub allocated: u128,
-    pub capacity: u128,
-}
-
 // Not really a DB model, just the result of a datastore function
```

Comment:
is this still applicable?

Reply:
Yes, because it doesn't correspond to any table schema